CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.0.10
```

Comment:
Since this adds a feature, it should be versioned as `2.1.0`

Reply:
...update the pubspec accordingly!

Reply:
Done